packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.scss

```diff
@@ -183,7 +185,11 @@
   flex-shrink: 0;
 }
 
-.dismissalOneline {
+.dismissOneline {
```

Comment:
Just checking, is this/will this be used anywhere? I don't see its usage in the .tsx.